src/os/bluestore/BlueStore.cc

```diff
@@ -7893,7 +7892,6 @@ void BlueStore::_kv_sync_thread()
       }
       for (auto txc : kv_submitting) {
 	assert(txc->state == TransContext::STATE_KV_QUEUED);
-	_txc_finalize_kv(txc, txc->t);
```

Comment:
This has to happen in kv_sync_thread when using ExtentFreelistManager.  For the bitmap one it could be moved.  Perhaps we should just drop the extent-based implementation....